compute/cloud-client/src/test/java/compute/SnippetsIT.java

```diff
@@ -230,4 +266,25 @@ public void testPatchFirewallRule() throws IOException, InterruptedException {
     }
   }
 
+  @Test
+  public void testInstanceOperations()
+      throws IOException, ExecutionException, InterruptedException {
+    Assert.assertSame(getInstanceStatus(MACHINE_NAME), Status.RUNNING);
+    StopInstance.stopInstance(PROJECT_ID, ZONE, MACHINE_NAME);
+    Assert.assertSame(getInstanceStatus(MACHINE_NAME), Status.TERMINATED);
+    StartInstance.startInstance(PROJECT_ID, ZONE, MACHINE_NAME);
+    Assert.assertSame(getInstanceStatus(MACHINE_NAME), Status.RUNNING);
+  }
+
+  @Test
+  public void testEncryptedInstanceOperations()
+      throws IOException, ExecutionException, InterruptedException {
+    Assert.assertSame(getInstanceStatus(MACHINE_NAME_ENCRYPTED), Status.RUNNING);
+    StopInstance.stopInstance(PROJECT_ID, ZONE, MACHINE_NAME_ENCRYPTED);
+    Assert.assertSame(getInstanceStatus(MACHINE_NAME_ENCRYPTED), Status.TERMINATED);
+    StartEncryptedInstance
+        .startEncryptedInstance(PROJECT_ID, ZONE, MACHINE_NAME_ENCRYPTED, RAW_KEY);
+    Assert.assertSame(getInstanceStatus(MACHINE_NAME_ENCRYPTED), Status.RUNNING);
+  }
```

Comment:
Each of these operations might take some time to finish.

Reply:
Added manual wait operation with a timeout.